ajax/libs/overpass/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "overpass",
+  "version": "3.0.2",
+  "filename": "overpass-webfont/overpass.css",
```

Comment:
You are right! Please add the `overpass-webfont/overpass.min.css`.
And you can use this tool(https://github.com/PeterDaveHello/web-minify-helper ) to minify the file.
You could refer to readme as well.